plugins/csp-wms-overlays/gui/css/csp-wms-overlays.css

```diff
@@ -0,0 +1,32 @@
+#wmsOverlays\.legendContainer {
```

Comment:
In my opinion using dots in container IDs is a bit unfortunate as it is easily confused with classes. I would suggest using dashes: `#wmsOverlays-legendContainer`. This requires several changes which I will not mark all in this review :smile: